modules/ztp-unboxing-factory-edge-cluster.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="configuring-edge-cluster-on-site_{context}"]
+= Configuring the edge cluster on site
+
+A custom UI is available for configuring the edge cluster on site:
```

Comment:
This should be a period-terminated sentence or a paragraph--Jupiter abstract guidance.

Reply:
sorry, need to remove this file too